src/index.html

```diff
@@ -5,8 +5,439 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com"
+      crossorigin="anonymous"
+    >
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a
+        class="header__link"
+        href="#home"
+      >
+        <img
+          src="images/logo.png"
+          alt="Moyo Logo"
+        >
+      </a>
+
+      <nav class="header__nav nav">
+        <ul class="nav__list">
+          <li class="nav__list-item">
+            <a
+              class="nav__link is-active"
+              href="#"
+            >
+             Apple
+            </a>
+          </li>
+
+          <li class="nav__list-item">
+            <a
+              class="nav__link"
+              href="#"
+            >
+             Samsung
+            </a>
+          </li>
+
+          <li class="nav__list-item">
+            <a
+              class="nav__link"
+              href="#"
+            >
+             Smartphones
+            </a>
+          </li>
+
+          <li class="nav__list-item">
+            <a
+              class="nav__link"
+              data-qa="nav-hover"
+              href="#"
+            >
+             Laptops&nbsp;&amp;&nbsp;Computers
+            </a>
+          </li>
+
+          <li class="nav__list-item">
+            <a
+              class="nav__link"
+              href="#"
+            >
+             Gadgets
+            </a>
+          </li>
+
+          <li class="nav__list-item">
+            <a
+              class="nav__link"
+              href="#"
+            >
+             Tablets
+            </a>
+          </li>
+
+          <li class="nav__list-item">
+            <a
+              class="nav__link"
+              href="#"
+            >
+             Photo
+            </a>
+          </li>
+
+          <li class="nav__list-item">
+            <a
+              class="nav__link"
+              href="#"
+            >
+             Video
+            </a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+    <main class="container">
+      <div class="card container__card" data-qa="card">
+        <img
+          class="card__image"
+          src="images/imac.jpeg"
+          alt="APPLE A1419 iMac 27'' Retina 5K Monoblock (MNED2UA/A)"
+        >
+
+        <p class="card__product-name">
+           APPLE A1419 iMac 27" Retina <br>
```

Comment:
Avoid using` <br>` to force line breaks or spacing. That's a job for CSS. If it's a paragraph, then make it a paragraph `<p>` or `<div>`